ringpop.go

```diff
@@ -362,6 +362,17 @@ func (rp *Ringpop) ringEvent(event interface{}) {
 	}
 }
 
+func (rp *Ringpop) GetMembers() []string {
+	members := rp.node.MemberStats().Members
```

Comment:
It seems slightly odd to depend on a Stats API to get the members? I'm pretty unfamiliar with the Go codebase. Is that the best API we've got from the Node type?

Also... I'd prefer that you did leak the 'Status' abstraction into the Ringpop type. Checking for 'Alive' and 'Suspect' shuld be limited to the Membership API.